index.html

```diff
@@ -2399,6 +2399,36 @@ <h4>Not Mapped</h4>
 					<span class="property not-mapped"><a href="#not_mapped">Not mapped*</a></span>
 				</td>
 			</tr>
+			<tr id="ariaBraillelabel">
+				<th><a class="property-reference" href="#aria-braillelabel"><code>aria-braillelabel</code></a></th>
+				<td class="attr-msaa-ia2">
+					<span class="property">Object Attribute: <code>braillelabel:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-uia">
+					<span class="property">Property: <code>AriaProperties.braillelabel</code>: <code>&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-atk">
+					<span class="property">Object Attribute: <code>braillelabel:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-axapi">
+					<span class="property">Property: AXBrailleLabel</span>
+				</td>
+			</tr>
+			<tr id="ariaBrailleroledescription">
+				<th><a class="property-reference" href="#aria-brailleroledescription"><code>aria-brailleroledescription</code></a></th>
+				<td class="attr-msaa-ia2">
+					<span class="property">Object Attribute: <code>brailleroledescription:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-uia">
+					<span class="property">Property: <code>AriaProperties.brailleroledescription</code>: <code>&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-atk">
+					<span class="property">Object Attribute: <code>brailleroledescription:&lt;value&gt;</code></span>
+				</td>
+				<td class="attr-axapi">
+					<span class="property">Property: AXBrailleRoleDescription</span>
+				</td>
+			</tr>
```

Comment:
I wonder if I should add a second entry for aria-brailleroledescription:

aria-roledescription is empty or whitespace characters
- IA2: not mapped
- UIA: not mapped
- ATK: not mapped
- AXAPI: not mapped

Reply:
or maybe it can just be a note?

Reply:
A note is fine. We also need to not expose when there is no aria-roledescription.